test/server/cards/01_SOR/LeiaOrganaAllianceGeneral.spec.js

```diff
@@ -45,48 +45,60 @@ describe('Leia Organa, Alliance General', function() {
             it('should allow passing the second attack', function () {
                 this.player1.clickCard(this.leiaOrgana);
                 this.player1.clickPrompt('Attack with a Rebel unit');
-                expect(this.player1).toBeAbleToSelectExactly([this.battlefieldMarine, this.fleetLieutenant, this.allianceXwing]);
-                expect(this.player1).not.toHaveEnabledPromptButton('Pass ability');
 
                 this.player1.clickCard(this.battlefieldMarine);
                 this.player1.clickCard(this.sundariPeacekeeper);
-                expect(this.battlefieldMarine.exhausted).toBe(true);
-                expect(this.sundariPeacekeeper.damage).toBe(3);
-                expect(this.battlefieldMarine.damage).toBe(1);
 
                 // second attack
                 expect(this.player1).toBeActivePlayer();
-                expect(this.player1).toBeAbleToSelectExactly([this.fleetLieutenant, this.allianceXwing]);
+                expect(this.player1).toBeAbleToSelectExactly([this.sabineWren, this.fleetLieutenant, this.allianceXwing]);
                 expect(this.player1).toHaveEnabledPromptButton('Pass ability');
                 this.player1.clickPrompt('Pass ability');
 
                 expect(this.player2).toBeActivePlayer();
                 expect(this.leiaOrgana.exhausted).toBe(true);
             });
-        });
-
-        describe('Leia\'s undeployed ability', function() {
-            beforeEach(function () {
-                this.setupTest({
-                    phase: 'action',
-                    player1: {
-                        groundArena: ['atst', { card: 'rebel-pathfinder', exhausted: true }],
-                        spaceArena: ['tieln-fighter'],
-                        leader: 'leia-organa#alliance-general'
-                    },
-                    player2: {
-                        groundArena: ['sundari-peacekeeper'],
-                        spaceArena: ['tie-advanced']
-                    }
-                });
-            });
 
-            it('can be activated with no target', function () {
+            it('should allow appropriate attack triggers to happen when either attack is declared', function () {
+                // unit with trigger first
                 this.player1.clickCard(this.leiaOrgana);
                 this.player1.clickPrompt('Attack with a Rebel unit');
+                this.player1.clickCard(this.sabineWren);
+                this.player1.clickCard(this.sundariPeacekeeper);
+
+                expect(this.sabineWren.damage).toBe(0);
+                expect(this.sundariPeacekeeper.damage).toBe(0);
+                expect(this.p2Base.damage).toBe(0);
+                expect(this.player1).toBeAbleToSelectExactly([this.sundariPeacekeeper, this.p1Base, this.p2Base]);
```

Comment:
Can you just add a comment explaining that this block is triggering the Sabine ability, since this test setup is kind of complex (and ditto below)